Bottom: 8f1cb4cc02401b6df11cdcc7ed72d59196fe15d1
Top:    c10e7a5c16ec82cb4ce80066914dc10ebad8680b
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-08 10:54:07 +0800

moving

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.h b/include/linux/vhost.h
similarity index 100%
rename from drivers/vhost/vhost.h
rename to include/linux/vhost.h
